recipe/meta.yaml

```diff
@@ -91,8 +91,11 @@ about:
   home: http://cairographics.org/
   dev_url: https://gitlab.freedesktop.org/cairo/cairo
   doc_url: https://www.cairographics.org/documentation/
-  license: LGPL 2.1 or MPL 1.1
-  license_file: COPYING
+  license: LGPL-2.1-only or MPL-1.1
+  license_file: 
+    - COPYING
+    - COPYING-LGPL-2.1
+    - COPYING-MPL-1.1
```

Comment:
@conda-forge/cairo, please make sure you're okay with these changes.  I think they are correct but it never hurts to have another pair of eyes on a thing like this.

Reply:
Yes, this change looks correct to me. Thanks!